packages/backend/src/config/config.production.ts

```diff
@@ -64,5 +64,9 @@ export function getProductionConfig(): Config {
         },
       },
     },
+    health: {
+      startedAt: getEnv('HEROKU_RELEASE_CREATED_AT'),
```

Comment:
How is this value exactly defined? I thought that we should just get date.now() when creating config and this will be correct.

Reply:
https://devcenter.heroku.com/articles/dyno-metadata#dyno-metadata

Reply:
Ok, going for date.now()